src/plugins/intel_gpu/src/kernel_selector/kernels/concatenation/concatenation_kernel_base.cpp

```diff
@@ -109,6 +109,11 @@ ConcatenationKernelBase::DispatchData ConcatenationKernelBase::SetDefault(const
     return dispatchData;
 }
 
+template<>
+bool KernelData::SkipKernelExecution<concatenation_params>(const concatenation_params& params, size_t kernel_id) {
+    return params.inputs[kernel_id].LogicalSize() == 0;
```

Comment:
Why concat need its own function?

Reply:
Because in concatenation each kernel has its own input and we must consider them separately